ext/pcre/php_pcre.c

```diff
@@ -1135,6 +1135,11 @@ static void php_do_pcre_match(INTERNAL_FUNCTION_PARAMETERS, int global) /* {{{ *
 		RETURN_FALSE;
 	}
 
+	if (start_offset == LONG_MIN) {
```

Comment:
Is it possible that `LONG_MIN` is smaller than the minimum value of `zend_off_t` on some platforms?

For 8.4 and master we could update the `start_offset` parameter of `php_pcre_match_impl()` to `ssize_t` since it's casted to `size_t` after negation, and use `SSIZE_MIN` here.

For stable branches we could use `ZEND_LONG_MIN` as `zend_long` is always the same as `zend_off_t`.